apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -661,15 +661,26 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
                 };
                 // If no primary value is set, set the first value as the primary value.
                 if (isEmpty(primaryValue) && !isEmpty(currentValues)) {
+                    const subAttributes: MultiValue[] = extractSubAttributes(EMAIL_ATTRIBUTE);
```

Comment:
Shall we move inside the if block of the email attribute as this is out of scope?

Reply:
Moved.